README.md

```diff
@@ -41,9 +53,10 @@ Its features comprise
 * IPv6
 * 6LoWPAN (RFC4944, RFC6282, and RFC6775)
 * UDP
-* RPL (storing mode)
+* RPL 
```

Comment:
I am wondering why we want to remove this information, instead of updating it (P2P-RPL)? If it's because it yields a better (simple) overview, or it "sells" better, then we should also remove the RFC tail following `6LoWPAN` two lines above.